MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -15,41 +15,184 @@ public class ServerConfiguration : BaseApplicationConfiguration
+        /// </summary>
+        public ServerConfiguration()
+        {
+            MetadataOptions = new[]
+            {
+                new MetadataOptions()
+                {
+                    ItemType = "Book"
+                },
+                new MetadataOptions()
+                {
+                    ItemType = "Movie"
+                },
+                new MetadataOptions
+                {
+                    ItemType = "MusicVideo",
+                    DisabledMetadataFetchers = new[] { "The Open Movie Database" },
+                    DisabledImageFetchers = new[] { "The Open Movie Database" }
+                },
+                new MetadataOptions
+                {
+                    ItemType = "Series",
+                    DisabledMetadataFetchers = new[] { "TheMovieDb" },
+                    DisabledImageFetchers = new[] { "TheMovieDb" }
+                },
+                new MetadataOptions
+                {
+                    ItemType = "MusicAlbum",
+                    DisabledMetadataFetchers = new[] { "TheAudioDB" }
+                },
+                new MetadataOptions
+                {
+                    ItemType = "MusicArtist",
+                    DisabledMetadataFetchers = new[] { "TheAudioDB" }
+                },
+                new MetadataOptions
+                {
+                    ItemType = "BoxSet"
+                },
+                new MetadataOptions
+                {
+                    ItemType = "Season",
+                    DisabledMetadataFetchers = new[] { "TheMovieDb" },
+                },
+                new MetadataOptions
+                {
+                    ItemType = "Episode",
+                    DisabledMetadataFetchers = new[] { "The Open Movie Database", "TheMovieDb" },
+                    DisabledImageFetchers = new[] { "The Open Movie Database", "TheMovieDb" }
+                }
+            };
+        }
 
         /// <summary>
         /// Gets or sets a value indicating whether to enable automatic port forwarding.
         /// </summary>
-        public bool EnableUPnP { get; set; }
+        public bool EnableUPnP { get; set; } = false;
 
         /// <summary>
         /// Gets or sets a value indicating whether to enable prometheus metrics exporting.
         /// </summary>
-        public bool EnableMetrics { get; set; }
+        public bool EnableMetrics { get; set; } = false;
 
         /// <summary>
         /// Gets or sets the public mapped port.
         /// </summary>
         /// <value>The public mapped port.</value>
-        public int PublicPort { get; set; }
+        public int PublicPort { get; set; } = DefaultHttpPort;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the http port should be mapped as part of UPnP automatic port forwarding.
+        /// </summary>
+        public bool UPnPCreateHttpPortMap { get; set; } = false;
```

Comment:
Can we put all this stuff in a NetworkingConfiguration class or something? `ServerConfiguration.cs` is starting to feel very bloated.

Reply:
Latest upload moves all network settings into a separate config file.

Am updating Network PR 2 to migrate settings across, and will submit a PR for the web.